lib/custom_lib/custom_lib.c

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2021, Legrand North America, LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <custom_lib.h>
+
+int custom_lib_get_value(int return_value_if_nonzero)
+{
+	int return_value = return_value_if_nonzero;
+
+	if (return_value == 0) {
+		return_value = CONFIG_CUSTOM_LIB_GET_VALUE_DEFAULT;
+	}
+
+	return return_value;
+}
```

Comment:
Can be simplified, e.g.
```suggestion
int custom_lib_get_value(int val)
{
	return val == 0 ? CONFIG_CUSTOM_LIB_GET_VALUE_DEFAULT : val;
}
```

Reply:
Accepted, and equivalent simplification manually rolled in.